src/components/Page.vue

```diff
@@ -41,6 +41,9 @@
           <li class="nav-item" v-if='modules.user_tasks'>
           <router-link class="nav-link" :to='{"name": "page.user_tasks", params: { group: group, cluster: cluster } }'>Cruise Control Tasks</router-link>
           </li>
+          <li class="nav-item" v-if='charts.enabled'>
```

Comment:
see below: lets use `modules.chart_page` (or similar name) inside `modules` to control the tab visibility ?